### PR TITLE
Add tail method to file module

### PR DESCRIPTION
**Pull request by joechainz**
Correct indenting issues from linter

### What does this PR do?
https://github.com/saltstack/salt/pull/50708/commits
### What issues does this PR fix or reference?
Fixes:

### Previous Behavior
Remove this section if not relevant

### New Behavior
Remove this section if not relevant

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [ ] Changelog
- [ ] Tests written/updated

### Commits signed with GPG?
Yes/No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

**Comment by Ch3LL** [comment]
it looks like you are trying to port https://github.com/saltstack/salt/pull/50708 to the master branch, but it looks like its not bringing over the changes to salt/modules/win_file.py and tests/integration/modules/test_file.py

Is there a reason for not including these changes?

**Review by Ch3LL — CHANGES_REQUESTED** [review]
still waiting on response to this comment: https://github.com/saltstack/salt/pull/56805#issuecomment-622402332

also this will need a test case added.

**Comment by mdschmitt** [comment]
Any update on this?

**Comment by dwoz** [comment]
@joechainz Are you still interested in getting this over the finish line?

File: salt/modules/file.py [hunk 1]
```diff
@@ -3551,6 +3551,73 @@ def touch(name, atime=None, mtime=None):
     return os.path.exists(name)
 
 
+def tail(path, lines):
+    """
+    .. versionadded:: Neon
```
Comment (twangboy) [inline salt/modules/file.py:3556]: Chlorine now, probably, maybe Argon

**Review by twangboy — CHANGES_REQUESTED** [review]

**Comment by twangboy** [comment]
Also needs a changelog and a test

**Comment by dwoz** [comment]
@joechainz Ping.

**Comment by dwoz** [comment]
This can be re-based and re-opened if it should get into `3008.x`

**Pull request closed without merging**